### ISSUE TITLE
Add auto tagging of listings

### ISSUE THREAD

**Issue by aaronarduino** [devict/job-board#29]
I think it might be nice to have auto tagging of jobs for things like crypto and web3 jobs. Then maybe users could filter or at least quickly see what type a job is.

**Comment by sethetter** [comment]
Yes, also for job specialties like frontend, backend, devops, data, product, etc.

**Comment by kevinfalting** [comment]
> auto tagging of jobs

interesting, how does something like that work?

**Comment by aaronarduino** [comment]
Off the top of my head I'm not sure how it would be implemented in Go but here is an example of what I was thinking in C#:

```c#
List<string> tags = new List<string>();
string content = (position + organization + description).ToLower();

if(content.Contains("web3"))
{
    tags.Add("web3");
} // else if ... etc... or wrap the if in a loop over a list of things we want to tag
```
It wouldn't be anything fancy and it might be wrong sometimes, but as long as we don't get too generic with the things we want auto tagged, it should be fine.

**Comment by sethetter** [comment]
I dig it! It probably makes sense to have a general tagging system in place as well. Whether that comes before this is optional though. Could start with this driven by a hard-coded list of tags first, and switch to having it driven by the user-generated tags later.
